### DISCUSSION TITLE
Using tags input in laravel livewire.

### DISCUSSION THREAD

**Discussion by penafarhan** [alpinejs/alpine#1102] [1. Help]
I just used and modified code from the example of tags input that provided in example folder [this](https://github.com/alpinejs/alpine/blob/master/examples/tags.html) Can anyone help me how to get the data from the input tags into the livewire. Thanks in advanced.

**Comment by penafarhan** [comment]
@calebporzio Hi sir, can you help for this?

**Reply by ryangjchandler** [reply]
@lordfarhan I think this is better in the Livewire discussion tab.

**Reply by penafarhan** [reply]
I think so, but no one answers my question about this in livewire forum. But thank you for your suggestion, sir.
